nerdlets/e2m-gui-nerdlet/form-components/rule-validator.js

```diff
@@ -152,6 +153,7 @@ RuleValidator.propTypes = {
   cardinalityTotals: PropTypes.object,
   cardinalityRuleViolation: PropTypes.bool,
   cardinalityAccountViolation: PropTypes.bool,
+  validatedFilterNRQL: PropTypes.string, // eslint-disable-line react/no-unused-prop-types
```

Comment:
curious why you are disabling the eslint check on this? is this prop not being used? It seems like you are using it.

Reply:
I am using it, as shown on line 23, but since I'm not using it explicitly (I never type props.validatedFilterNRQL) it doesn't recognize this usage and the lint fails so I had to override this check. But yes it is used.

Reply:
👍